### PR TITLE
Updated Footer

### LINKED ISSUE #391: Footer
Similar to ZotHacks 2024: footer needs links to
- Hack (https://hack.ics.uci.edu)
- Email (irvinehacks2025@gmail.com)
- Discord (https://discord.com/invite/pvkGxq2AWM)
- Instagram (https://www.instagram.com/hackatuci/)
- Tiktok (https://www.tiktok.com/@hackatuci)

with its UI updated using [Figma](https://www.figma.com/design/CyxJ0OSjgNPwb0eeTgq84s/IrvineHacks-2025?node-id=1-2&node-type=frame&m=dev) as reference

### PR DESCRIPTION
**Pull request by cyriljoby**
Resolves #391 

Created footer with icons and links to Hack's instagram, tiktok, mail and discord.
Updated the heart icon to reflect this year's design.

### CODE REVIEW

File: apps/site/src/lib/components/Footer/Footer.module.scss
Comment (waalbert) [inline apps/site/src/lib/components/Footer/Footer.module.scss:3]: Chore: remove unused comment

File: apps/site/src/lib/components/Footer/Footer.tsx
Comment (waalbert) [inline apps/site/src/lib/components/Footer/Footer.tsx:56]: Due to Student Center regulations, we have to specify that IrvineHacks is hosted by us, Hack at UCI, so switch the text of the footer to say "Made with ❤️ by Hack at UCI ..."

And that means we also have to add our Hack logo linked to our club site.

**Review by waalbert — CHANGES_REQUESTED** [review]
Awesome work Cyril! Here are a few requested changes

File: apps/site/src/assets/logos/discord.svg
Comment (taesungh) [inline apps/site/src/assets/logos/discord.svg:1]: This does not appear to be the official Discord logo. Please make sure to use the official version (e.g. `discord-mark-white.svg`) from [Discord's Branding Guidelines](https://discord.com/branding).

File: apps/site/src/assets/logos/tiktok.svg
Comment (taesungh) [inline apps/site/src/assets/logos/tiktok.svg:1]: Same note here: this does not appear to be an official TikTok logo, please use `TikTok-Social-Icon-Rounded-Square-Mono-Black.svg` from the [TikTok Brand Guidelines](https://tiktokbrandbook.com).

File: apps/site/src/lib/components/Footer/Footer.tsx
Comment (waalbert) [inline apps/site/src/lib/components/Footer/Footer.tsx:78]: Chore: remove unused comment

**Review by waalbert — CHANGES_REQUESTED** [review]
Some more things

File: apps/site/src/lib/components/Footer/Footer.tsx
Comment (taesungh) [reply apps/site/src/lib/components/Footer/Footer.tsx:19]: @waalbert, could you set up forwarding on a vanity email instead, e.g. contact@irvinehacks.com ?

Comment (waalbert) [reply apps/site/src/lib/components/Footer/Footer.tsx:19]: Yes, we will use the vanity email instead.

**Review by waalbert — DISMISSED** [review]
Thanks for the work Cyril! Don't merge until the new Landing Page is completed

**Review by waalbert — APPROVED** [review]

**Pull request merged**
